chapters/VK_EXT_debug_utils.txt

```diff
@@ -44,6 +44,14 @@ include::../api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
 include::../validity/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
+.Valid Usage
+****
+  * pname:pNameInfo\->objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
+  * pname:pNameInfo\->objectHandle must: not be dlink:VK_NULL_HANDLE
```

Comment:
Could be.  This was how it was with the Debug Marker version.  I could remove it since a valid handle seems to imply non-NULL.  But I didn't want to assume that without looking through the spec first.  Where I see VK_NULL_HANDLE being used, it does seem to be an independent value from "a valid handle of type XXX".  So, if we do state that it must be a valid handle, I think that's good enough.  :+1: